demos-europe/demosplan-core
# bug(refs T30859): Add permission request for choosing county

**Ticket:** https://yaits.demos-deutschland.de/T30859

When creating a new statement, you should only in robobsh and regio be able to choose a county. 

### How to review/test
Login as Privatperson, go to a procedure and click on "Reden Sie mit". The option "Einen Kreis direkt auswählen" should only be available in robob and regio.

### Linked PRs (optional)

- https://github.com/demos-europe/demosplan-project-robobsh/pull/14
- https://github.com/demos-europe/demosplan-project-regio/pull/17
- https://github.com/demos-europe/demosplan-project-bobsh/pull/15
